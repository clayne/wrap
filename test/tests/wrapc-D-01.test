wrapc | /dev/null | -D+ | wrapc-D-01.txt | 0
