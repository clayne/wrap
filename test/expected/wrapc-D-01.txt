  ? As a special-case, if the first line is NOT a comment, then just wrap all
  ? lines using the leading whitespace of the first line as a prototype for +
  ? all subsequent lines, i.e., do NOT ever tell wrap(1) to pass text through
  ? verbatim (below).
